DirectXTex/DirectXTex_Desktop_2022.vcxproj

```diff
@@ -170,6 +170,8 @@
       <ExternalWarningLevel>Level4</ExternalWarningLevel>
       <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
       <SupportJustMyCode>false</SupportJustMyCode>
+      <GuardEHContMetadata>
```

Comment:
Is this already defaulted correctly in VS 2022?

Reply:
Didn't found any information about the it been default on VS 2022. I don't think it would be since enabling it requires your dependencies to also have it enabled which would break projects that consume precompiled binaries.

Reply:
It appears to be "undefined" by default, neither true or false. Explicit adding it to Release|x64 and Release|ARM64 (and maybe Profile|x64/arm64) seems reasonable.